project/.github/FUNDING.yml.jinja

```diff
@@ -0,0 +1,4 @@
+github: {{ author_username }}
+ko_fi: {{ author_username }}
+custom:
+- https://www.paypal.me/{{ author_username }}
```

Comment:
We could add questions in `copier.yml`. However I feel like it's easier to let users generate a project, commit everything, then remove the few things they don't want, and future updates will keep these things removed.

However I agree that funding makes less sense here than on my Python projects. Maybe we could register the shellm-org on GitHub sponsors, and list it here instead of the author username? That would be a bit less fair for external users using the template though, as by default it would redirect *their* funding to *our* org :sweat_smile: 

Maybe we can just remove the funding file for now.

Reply:
Alright! 👍🏻